pkg/cache/configmaps.go

```diff
@@ -69,12 +69,12 @@ func RegisterConfigMapUpsertCallback(volID, cmID string, f func(key, value inter
 	}
 	configmapUpsertCallbacks.Store(volID, f)
 	ns, name, _ := SplitKey(cmID)
-	cm := client.GetConfigMap(ns, name)
-	if cm != nil {
-		f(BuildKey(cm.Namespace, cm.Name), cm)
-	} else {
-		klog.Warningf("not found on get configmap for %s vol %s", cmID, volID)
+	cm, err := client.GetConfigMap(ns, name)
```

Comment:
I do think this makes sense now that https://issues.redhat.com/browse/BUILD-965 means the driver might not be able to read the secret

In fact, I would change the error signature of this method to return and error, and propagate that error up the call stack

Seems like that was missed when https://issues.redhat.com/browse/BUILD-965 was implemented